tailwind.config.js

```diff
@@ -2,7 +2,15 @@
 module.exports = {
   content: ["./src/**/*.{html,js}"],
   theme: {
-    extend: {},
```

Comment:
that is great thing, All things working well

Reply:
good work